app/models/theater.rb

```diff
@@ -1,4 +1,9 @@
 class Theater < ApplicationRecord
   validates :name, presence: true, uniqueness: true, length: { maximum: 255 }
   has_many :plays, dependent: :destroy
+
+  # Returns a list of the theater's plays for the current voting period
+  def current_voting_period_plays
```

Comment:
For these kinds of methods, can you look into ActiveRecord _scopes_ please? Scopes will work similarly to class methods, but they have some unique benefits.